payroll_rule_time_parameter/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Payroll Rule Time Parameter",
+    "summary": "",
+    "author": "nimarosa, appstogrow, Odoo Community Association (OCA)",
+    "category": "Human Resources",
```

Comment:
This should probably be "Payroll"

Reply:
Yes you are right. Changing it in next commit.